improver_tests/standardise/test_AdjustLandSeaPoints.py

```diff
@@ -335,9 +359,9 @@ def test_with_regridding(self):
     def test_multi_realization(self):
         """Test that the expected changes occur and meta-data are unchanged
         when handling a multi-realization cube."""
-        cube = self.cube.copy()
-        cube.coord("realization").points = [1]
-        cubes = iris.cube.CubeList([self.cube, cube])
+        cube1 = add_coordinate(self.cube, [0], "realization")
+        cube2 = add_coordinate(self.cube, [1], "realization")
+        cubes = iris.cube.CubeList([cube1, cube2])
```

Comment:
Can use `cube = add_coordinate(self.cube, [0, 1], "realization", units=1)`.

Reply:
Done. Note the argument is coord_units rather than units.